zbus/src/object_server/interface/mod.rs

```diff
@@ -60,8 +77,9 @@ pub trait Interface: Any + Send + Sync {
         property_name: &'call str,
         value: &'call Value<'_>,
         emitter: &'call SignalEmitter<'_>,
+        header: &'call message::Header<'_>,
```

Comment:
While this currently can always get a header passed to it, this will hopefully be changing at some point: https://github.com/dbus2/zbus/issues/898 . So best to make this optional as well. It will also be more consistent that way.